README.md

```diff
@@ -29,10 +29,10 @@ Corollary: the activity of this repos reflects the engagement of the community a
 
 ## Expansions of Creed - Culture comes first
 
-### [I will never stop learning](#i-will-never-stop-learning)
+### [We will never stop learning](#i-will-never-stop-learning)
 The first and most important line of the creed means that you’re never finished. There is always more to learn. The enemy of this is thinking you’re right, putting too much weight on your experience, or passing up an opportunity to understand someone’s point of view.
 
-### [I will build our business  through creating the most value for our community of users, customers and collegues.](#i-will-build-our-business-through-creating-the-most-value-for-our-community-of-users-customers-and-colleagues)
+### [We will build our business  through creating the most value for our community of users, customers and collegues.](#i-will-build-our-business-through-creating-the-most-value-for-our-community-of-users-customers-and-colleagues)
```

Comment:
Probably, we should also update the markdown links: instead of (#i-will...), it should be (#we-will).